presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -915,7 +905,7 @@ private PlanBuilder limit(PlanBuilder subPlan, Optional<OrderBy> orderBy, Option
                 .collect(toImmutableMap(expression -> expression, builder::translate));
     }
 
-    private static SortOrder toSortOrder(SortItem sortItem)
```

Comment:
why not to move this method to `SortItem`?

Reply:
SortItem is in the parser module, so it can't see SortOrder.